packages/smart-wallet/test/supports.js

```diff
@@ -117,17 +117,24 @@ export const makeMockTestSpace = async log => {
   produce.zoe.resolve(zoe);
   produce.feeMintAccess.resolve(feeMintAccessP);
 
-  const vatLoader = name => {
+  /** @type {VatLoader} */
+  const vatLoader = async name => {
+    /** @type {unknown} */
+    let vatP;
     switch (name) {
-      case 'mints':
-        return mintsRoot();
+      case 'mints': {
+        vatP = mintsRoot();
```

Comment:
why change the return to setting a value to return outside the switch?

Reply:
I was trying to have as little casting goo as possible.  A local `@typedef` serves better.